packages/pds-ember/app/styles/pds/core/index.scss

```diff
@@ -1,2 +1 @@
-@use "links";
```

Comment:
🚨 This is an intentional breaking change. The `pds/core` module will be dissolved over the course of the next few PRs.